.github/workflows/docker-builds.yaml

```diff
@@ -0,0 +1,75 @@
+name: Build prefect docker images
+
+# Note: Conda support for 3.11 is pending. See https://github.com/ContinuumIO/anaconda-issues/issues/13082
+
+on:
+  # On each commit merged into main, push sha and branch patterns
+  push:
+    branches: main
+  # On workflow_dispatch, push sha and branch patterns
+  workflow_dispatch:
+
+jobs:
+  publish-docker-images:
+    name: Build Docker images
+    runs-on: ubuntu-latest
```

Comment:
I believe you'll need "environment: dev" like we do for the prod environment https://github.com/PrefectHQ/prefect/blob/main/.github/workflows/release.yaml#L186

Reply:
good catch! https://github.com/PrefectHQ/prefect/pull/7900/commits/d6defda08b8d0224e9347ca0aafd9b66044a9364